src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,55 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+
+    private static final int MAX_SIZE = 10;
+    private static final String PUT_EXCEPTION_MESSAGE =
+            "Can't add new data, the storage is full";
+    private K[] keys;
+    private V[] values;
+    private int itemsInStorage = 0;
```

Comment:
this field will be initialized as 0 by default
also better to rename it. This field represents size of your storage, so let's name it like that
```suggestion
    private int itemsInStorage;
```